### PR TITLE
Add more folder names for Kubernetes

### LINKED ISSUE #2134: [Icon Request] Support ".k8s" folders to use the existing "k8s" folder icon
- [ x] I'm sure this issue is not a _duplicate_.
- [x ] I'm submitting an icon request
  
The "k8s" folders use a nice kubernetes icon. Recently we have renamed all of our "k8s" folders into ".k8s". This has moved all the ".k8s" folders to the top of their parent containers, however we appear to have lost the icon associated with the "k8s" folder.

It would be great to get the icon back for improved visibility.

### PR DESCRIPTION
**Pull request by mmisztal1980**
_**Fixes #2134 **_

**Changes proposed:**

- [x] Add
- [ ] Delete
- [ ] Fix
- [ ] Prepare

Sometimes developers wish to have their k8s folders at the top of their folder structure for improved visibility. When prefixing the folders with a dot, the vscode-icons' icon is lost.

This PR adds the **kubernetes** folder icon to following folders:
- .k8s
- .kube

### CODE REVIEW

**Comment by mmisztal1980** [comment]
> Since you are doing this can you please add also the `dot` variant for the other naming?

@JimiC remaining folders prefixed, as requested

File: src/iconsManifest/supportedFolders.ts [hunk 1]
```diff
@@ -229,7 +229,16 @@ export const extensions: IFolderCollection = {
...
-      extensions: ['kubernetes', 'k8s', 'kube', 'kuber'],
+      extensions: [
+        'kubernetes', 
+        'k8s',        
+        'kube',        
+        'kuber',
+        '.kubernetes',
+        '.k8s',
+        '.kube',
+        '.kuber'
```
Comment (JimiC) [inline src/iconsManifest/supportedFolders.ts]: Trailing comma missing.

Comment (mmisztal1980) [reply src/iconsManifest/supportedFolders.ts]: fixed

**Comment by mmisztal1980** [comment]
@JimiC is there anything else required in order for this to be merged?

**Comment by JimiC** [comment]
Only your patience

**Pull request merged**